examples/waspleau/ext/workers/utils.js

```diff
@@ -0,0 +1,11 @@
+import Prisma from '@prisma/client'
+
+const prisma = new Prisma.PrismaClient()
```

Comment:
Yeah, we should expose Entities a bit better somehow. The trick in Jobs acting similar to Operations is I _think_ we may hit snags with the serialization/deserialization process. Any args we pass into jobs will need to be serialized by the executor (in this case pg-boss to PostgreSQL). It should be fine for simpler types but unsure how it would be for something complex.

The other option is we somehow expose via imports. I will create an issue and post in Discord for discussion. Thanks

Reply:
Yep, imports are always an option but are not so cool :D!

Regarding serialization -> but we are also doing serialization for Operations! Because they are happening over the network. So actually it wouldn't be a problem but a common thing for both? Maybe an additional reason to consider somehow connecting them more?

Reply:
Haha yeah, not so cool.

Yes, we are for the arguments to operation calls, but not really the Entities, right? I'm thinking we'd have trouble if we tried to serialize the `context` (so they could access entities like from Ops args).

Reply:
But you are correct, one fallback is to create an Action, and they just make the call to the action where they would have normally needed the entity itself.

Reply:
Ah got it now, so in this case all args need to be serialized - ok ok a bit tricky!